stack.yaml

```diff
@@ -1,6 +1,7 @@
 resolver: lts-10.0
 packages: [.]
-extra-deps: []
+extra-deps:
+- time-units-1.0.0
```

Comment:
Unfortunately, `time-units` package is not in LTS :( Adding this `extra-dep` won't allow us to be in LTS anymore which is sad... So let's remove this dependency. And use `Integer` data type instead of type safe `time-units`

Reply:
Probably we could make newtypes instead, because it is easy to mess up with the same type but different meanings? :thinking:

Reply:
@vrom911 You're 146% right. It's really a shame that such package like `time-units` is not on stackage... Having newtypes for such problem is the right solution. I agree with you here, let's introduce our own newtypes.

Reply:
I would like to wait the results of the discussion in slack about what to do with issue before applying newtypes.. Newtypes is not the best option in this case.. :(

Reply:
Ping.

Reply:
@gromakovsky We started work on `safe-time` library. Join #safe-time channel in Slack.